pepper-apis/studybuilder-cli/studies/osteo/patches/osteo-consent-parental-v3.conf

```diff
@@ -0,0 +1,115 @@
+{
+  "pdfs": [
+    {
+      "name": "osproject-consent-parental-v2",
```

Comment:
Name and file name has v2 !

Reply:
@ssettipalli - Right!  The values for name and filename are the **existing** values deployed.  We debated updating the name to be v3, removing the version, or leaving it as is.  Is there a preference for this?  I think I've seen all three ways in the last couple of sprints.

Reply:
For the pdf PR's I did I didn't have the -v* in the names. I felt like its cleaner that way. 
Here since its already there (-v2), I thought you might wanted to update it but may be missed. 
If its intentional, thats fine with me. Either way, I am ok.